collectors/python.d.plugin/edgex/edgex.chart.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+# Description: EdgeX Platform netdata python.d module
+# Author: Odysseas Lamtzidis (OdysLam)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Module Version: 0.2
+
+import json
+import threading
+
+from collections import namedtuple
+
+try:
+    from Queue import Queue
+except ImportError:
+    from queue import Queue
```

Comment:
Unless you're using some third-party module here that happens to implement the same API as the standard library's `queue` module, you can drop this. It's bee lowercase since at least Python 2.7 and for all of 3.x, and we don't support anything older than 2.7.

Reply:
No so fast guys

```
ilyam@debian-s-1vcpu-1gb-fra1-01:~$ python
Python 2.7.13 (default, Sep 26 2018, 18:42:22)
[GCC 6.3.0 20170516] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from queue import Queue
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named queue
>>> from Queue import Queue
>>>
```

Reply:
it is there for a reason

Reply:
You are absolutely right @ilyam8, tested it during the plugin dev. 
Sorry for jumping to conclusions.

Reply:
Ah, my bad. Misread the 2.7.0 documentation and didn't see the use of the uppercase 'Q' there.